public/docs/dart/latest/guide/component-styles.jade

```diff
@@ -1 +1,40 @@
-!= partial("../../../_includes/_ts-temp")
\ No newline at end of file
+extends ../../../ts/latest/guide/component-styles.jade
+
+block includes
+  include ../_util-fns
+
+//- TODO: consider adding material equivalent to TS Appendices 1 & 2 if relevant.
+
+block style-url-alert
+  :marked
+    The URL is ***relative to the Dart application package root*** which usually
```

Comment:
`styleUrls` paths worked relative to the component file until beta.17 for me in Dart